Algorithms_and_Data_Structures/Maths/reverse_integer.py

```
@@ -0,0 +1,30 @@
+def input_array(arr, n):  # Not needed for single integer, but keeping format
```

Ruff (0.12.2)

Unused function argument: `n`

(ARG001)